_issues/trade.md

```diff
@@ -0,0 +1,30 @@
+---  
+layout: post  
+title: "But isn't she for free trade?"  
+categories: policy
+---  
+
+Q: Doesn't Hillary support trade agreements that take jobs away from American workers?
+  
+A: Not at all. Despite what Donald Trump would have you believe, the world is a complicated place. Trade deals are complex, because they are supposed to benefit all of the nations involved. Hillary believes that if a trade deal does not benefit American workers, it must be opposed.  
+
+Consider the facts:
+
+* Fact: Hillary is neither pro-trade nor anti-trade. Instead, she evaluates each trade deal on its merits and determines the overall benefit to American workers. As [Gene Sperling, director of the National Economic Council in the Clinton and Obama administrations](http://www.politifact.com/truth-o-meter/statements/2016/apr/10/bernie-s/clinton-voted-virtually-every-trade-agreement-kill/) put it:
```

Comment:
maybe change initial sentence to "Hillary is neither **wholly** pro- or anti- **free** trade." ?